src/Infrastructure/Identity/ApplicationUser.cs

```diff
@@ -12,4 +12,7 @@ public class ApplicationUser : IdentityUser
     public DateTime RefreshTokenExpiryTime { get; set; }
 
     public string? ObjectId { get; set; }
+
+    public bool IsRootUser { get; set; }
+    public bool IsTenantUser { get; set; }
```

Comment:
What does this mean?

Reply:
because root user must not be removed

Reply:
It is useful Marching the user who was that he was that for that tenant who administers to not be able to cancel him or that the administrative permits removed

Reply:
I can't make anything of that sentence... 

What is a TenantUser?